M0narc/fast_api
# post endpoint for creatign todos

## Description

post endpoint for creating new todos in the db

## Testing Instructions

using postman or swagger create a new todo list in the db using the new endpoint
and then check if it's there using the read_all endpoint

## Screenshots



![imagen](https://github.com/M0narc/fast_api/assets/46531321/607cd025-ddb1-48be-b8c6-14a2eb164542)
